src/ts/index.ts

```diff
@@ -1 +1,8 @@
 import "../css/style.css";
+import {enableGoToTopBtn} from "./buttons/goToTop";
+import {MovieHandler} from "./handlers/MovieHandler";
+
+window.onload = (): void => {
+    enableGoToTopBtn();
+    new MovieHandler();
+}
```

Comment:
This is pretty neat! Great work! 

The only comment I really have here about the design is around instantiating `new MovieHandler()` to have the side-effect of 'starting' up the app. It definitely works though. We can chat about general design principles, but overall this looks good!